index.html

```diff
@@ -16,6 +16,56 @@
     <script type="text/javascript" src="js/application.js"></script>
 </head>
 <body>
+    <div class="container">
+        <div class="row">
+                <h1>Minesweeper Reborn</h1>
+                <h3>Avoid the bomb and flag them or you'll get blown into pieces.</h3>
+        </div>
+        <div>
+            <table id="open-board" class="table table-bordered">
+               <tbody id="tableBody">
+                    <tr>
+                        <td></td>
```

Comment:
This works, but if you want a bigger mine or a dynamic size mine, you should generate this dyanmically